src/components/fields/SchemaField.js

```diff
@@ -91,7 +91,7 @@ function DefaultTemplate(props) {
   }
 
   return (
-    <div className={classNames}>
+    <div id={id} className={classNames}>
```

Comment:
* is this the type of solution you had in mind? 
* should it be configurable from api?